R/utils.R

```diff
@@ -16,6 +16,16 @@ parse_answer = function(api_answer, multiple = FALSE) {
 
   xml_source = content(api_answer, encoding = "ISO-8859-1")
 
+  apicontrol = xml_text(xml_find_all(xml_source, "//apicontrol"))
+
+  if (apicontrol == "invalid") {
```

Comment:
Ha, I was just thinking about this before lunch. I don't think this is necessary because `parse_answer` is not exported so it should only receive arguments that we control. And we don't propose any way to generate invalid queries as far as I can tell.

So I believe this is just a case of [Pokémon Exception Handling](https://blog.codinghorror.com/new-programming-jargon/).

Reply:
Yep. The only thing that can happen is invalid API key.